eng/scripts/Language-Settings.ps1

```diff
@@ -224,6 +229,12 @@ function UpdateDocsMsPackages($DocConfigFile, $Mode, $DocsMetadata) {
 
   $outputPackages = @()
   foreach ($package in $packageConfig.npm_package_sources) {
+    # If the package is in the exclusion list, remove it
+    if ($PackageExclusions.ContainsKey((Get-PackageNameFromDocsMsConfig $package.name))) {
```

Comment:
Lets store the value of Get-PackageNameDocsMsConfig in this loop so we don't call it so many times.